def handle_exc(exc):
    pass
